<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressBookDialog</class>
 <widget class="QDialog" name="AddressBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableViewAddressList">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>621</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonAddressBookCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>195</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameAddressList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>641</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 2px solid #C0C0C0;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frameAddressList</zorder>
  <zorder>tableViewAddressList</zorder>
  <zorder>pushButtonAddressBookCancel</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
